show/main.py

```diff
@@ -1227,10 +1227,23 @@ def users(verbose):
 
 @cli.command()
 @click.option('--since', required=False, help="Collect logs and core files since given date")
+@click.option('--force', is_flag=True, help="Enable hardware register dump even with system interruption")
+@click.option('--yes', is_flag=True, help="Enable force generating dump option")
```

Comment:
Suggest changing help message to: `Assume "yes" as answer to all prompts and
    run non-interactively`